File: commands/linked.js
const { Command, Middleware } = require('yamdbf');
const { resolve, expect } = Middleware;
const utils = require('../utils');

module.exports = class extends Command {
	constructor() {
		super({
			name: 'linked',
			desc: 'Check if account is linked to a forum profile',
			usage: '<prefix>linked',
		});
	}

	action(message) {
		// Save to database & send forum PM to user
        utils.post('linked.php', { discord_id: message.author.id, }, body => {
            if (body == 'success') {
                // Respond with confirmation
                message.reply('No worries! Your Discord Account is already linked to your Forum Profile.');
            } else {
                // Respond with error
                console.log(body);
                message.reply('Sorry! Your Discord Account is not yet linked to your Forum Profile. Use `!link <forum_username>` to get started.');
            }
        });
	}
};

File: commands/linked.py
from commands.base_command  import BaseCommand
import requests
import settings
from utils                  import get_base_url, get_emoji

class Linked(BaseCommand):
    def __init__(self):
        description = "Check if your account is linked to a forum profile"
        params = []
        super().__init__(description, params)

    async def handle(self, params, message, client):
        url = get_base_url() + 'linked.php'
        data = {
            'discord_id': message.author.id,
        }
        r = requests.post(url, data)
        
        if r.text == 'success':
            msg = 'No worries! Your Discord Account is already linked to your Forum Profile.'
        else:
            msg = 'Sorry! Your Discord Account is not yet linked to your Forum Profile. Use `' + settings.COMMAND_PREFIX + 'link <forum_username>` to get started.'
        
        await client.send_message(message.channel, msg)

